src/Compiler/Checking/PostInferenceChecks.fs

```diff
@@ -521,7 +521,7 @@ let AccessInternalsVisibleToAsInternal thisCompPath internalsVisibleToPaths acce
         accessSubstPaths (thisCompPath, internalsVisibleToPath) access)
 
 
-let CheckTypeForAccess (cenv: cenv) env objName valAcc m ty =
+let CheckTypeForAccess (cenv: cenv) env objName valAcc isCompilerGenerated m ty =
```

Comment:
Sorry for the nitpick, but should this be named something like `accessedValIsCompilerGenerated` to make it clear that it is the value being accessed in the `valAcc` that is compiler generated and not the `ty`?